aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/AppSyncResponseDeserializerTest.java

```diff
@@ -61,7 +61,7 @@ public void awsDateTypesCanBeDeserialized() {
                 .build();
         Boolean deleted = false;
         Integer version = 42;
-        Long lastChangedAt = Long.valueOf(1594858827);
+        Temporal.Timestamp lastChangedAt = new Temporal.Timestamp(1594858827, TimeUnit.SECONDS);
```

Comment:
You could make a `Temporal.timestamp(long, TimeUnit)` util, too, that might be cool.

Reply:
Agreed, that would be nice.  Maybe in a subsequent PR.